be/src/column/binary_column.cpp

```diff
@@ -172,11 +172,17 @@ void append_fixed_length(const Buffer<Slice>& strs, Bytes* bytes, typename Binar
 
     size_t offset = bytes->size();
     bytes->resize(size + copy_length);
-    for (const auto& s : strs) {
-        strings::memcpy_inlined(&(*bytes)[offset], s.data, copy_length);
-        offset += s.size;
-        offsets->emplace_back(offset);
+
+    size_t rows = strs.size();
+    size_t length = offsets->size();
+    raw::stl_vector_resize_uninitialized(offsets, offsets->size() + rows);
+
+    for (size_t i = 0; i < rows; ++i) {
+        memcpy(&(*bytes)[offset], strs[i].get_data(), copy_length);
```

Comment:
why not use `strings::memcpy_inlined`?

Reply:
for const literal size (copy_length). buildin_memcpy will got a better performance

example: https://godbolt.org/z/oKaW3sW7b

```
  41.36%  starrocks_be      [.] starrocks::BinaryDictPageDecoder<(starrocks::LogicalType)17>::next_batch
  14.71%  starrocks_be      [.] starrocks::append_fixed_length<unsigned int, 8ul>
   7.67%  starrocks_be      [.] strings::memcpy_inlined
```

```
literal_memcpy(Object*, Object*):
        vmovdqu xmm0, XMMWORD PTR [rdi]
        vmovdqu XMMWORD PTR [rsi], xmm0
        ret
```